perf/flame.jl

```diff
@@ -17,6 +17,7 @@ simulation = CA.get_simulation(config)
 
 import SciMLBase
 SciMLBase.step!(integrator) # compile first
+SciMLBase.step!(integrator) # compile print_walltime_estimate, which skips the first step to avoid
```

Comment:
It looks like that the comment has been truncated midsentence?

Reply:
Yeah, will fi